components/base/kinematicbase/differentialDrive.go

```diff
@@ -332,11 +332,11 @@ func (ddk *differentialDriveKinematics) ErrorState(
 	if len(plan) < 2 {
 		return nil, errors.New("diff drive motion plan must have at least two waypoints")
 	}
-	nominalPose, err = ddk.executionFrame.Transform(plan[currentNode])
+	nominalPose, err = ddk.planningFrame.Transform(plan[currentNode])
```

Comment:
This is currently a bug, the execution frame will be a 3DOF frame which wne called with a position only plan will try to transform a frame with only 2DOF